main/zoe/guide/README.md

```diff
@@ -104,19 +104,34 @@ Writing smart contracts that run on Zoe is easy, but let's look
 at a simple contract. This contract only does one thing, and
 it's pretty useless - it gives you back what you put in. Let's call it
 `automaticRefund`. Let's say the code of `automaticRefund` looks like this (see
-the [real contract code here](https://github.com/Agoric/ERTP/blob/master/core/zoe/contracts/automaticRefund.js)):
+the [real contract code here](https://github.com/Agoric/agoric-sdk/blob/master/packages/zoe/contracts/automaticRefund.js)):
 
 ```js
 export const makeContract = (zoe, terms) => {
-  return {
-    instance: {
-      makeOffer: async escrowReceipt => {
-        const { offerHandle } = await zoe.burnEscrowReceipt(escrowReceipt);
-        zoe.complete([offerHandle]);
+  let offersCount = 0;
```

Comment:
Thanks so much for updating this! I think we should leave out the `offersCount` and related code (`getOffersCount`, `offersCount += 1`) to simplify the contract as much as possible. I put `offersCount` in there for testing purposes, but it's probably not needed here.